app/src/main/java/com/hedvig/app/feature/home/ui/claimstatus/data/PillData.kt

```diff
@@ -0,0 +1,75 @@
+package com.hedvig.app.feature.home.ui.claimstatus.data
+
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import com.hedvig.android.owldroid.graphql.HomeQuery
+import com.hedvig.android.owldroid.type.ClaimOutcome
+import com.hedvig.android.owldroid.type.ClaimStatus
+import com.hedvig.app.R
+import com.hedvig.app.ui.compose.theme.ClaimStatusColors
+
+data class PillData(val text: String, val type: PillType) {
+
+    sealed class PillType {
+        data class Contained(val color: Color) : PillType()
+        object Outlined : PillType()
+    }
+
+    companion object {
+        @Composable
```

Comment:
Doesn't it become a little bit architecturally weird to have a `data class`-factory function that is `@Composable`? 😬